leapp/utils/output.py

```diff
@@ -8,37 +8,44 @@
 from leapp.exceptions import LeappRuntimeError
 
 
-def _get_colors():
-    if sys.stdout.isatty():
-        return "\033[1;31m", "\033[0;0m"
-    return "", ""
+class Color(object):
+    reset = "\033[0m" if sys.stdout.isatty() else ""
+    bold = "\033[1m" if sys.stdout.isatty() else ""
+    red = "\033[1;31m" if sys.stdout.isatty() else ""
+    green = "\033[1;32m" if sys.stdout.isatty() else ""
 
 
-def report_errors(errors):
-    if errors:
-        red, reset = _get_colors()
-        sys.stdout.write("""{red}
-============================================================
-                        ERRORS
-============================================================{reset}\n\n""".format(red=red, reset=reset))
-        for error in errors:
-            print_error(error)
-        sys.stdout.write("""{red}
-============================================================
-                     END OF ERRORS
-============================================================{reset}\n\n""".format(red=red, reset=reset))
+def pretty_block(string, color=Color.bold, width=60):
+    return (color + "=" * width + "\n"
+            + string.center(width) + "\n"  # noqa: W503
+            + "=" * width + Color.reset + "\n\n")  # noqa: W503
```

Comment:
As we suggest to everyone to not use concatenation of strings in this way, we should not do that as well. Anyway, this is much readable for me:

```
return "{color}{separator}\n{text}\n{separator}{reset}\n\n".format(
    color=color,
    separator="="*width,
    reset=reset,
    text=string.center(width)
```

Reply:
Replaced.